draft-ietf-ppm-dap.md

```diff
@@ -2726,11 +2666,6 @@ privacy is a strong privacy definition, and protects users in extreme
 circumstances: even if an adversary has prior knowledge of every input in a
 batch except for one, that one record is still formally protected.
 
-[OPEN ISSUE: While parameters configuring the differential privacy noise (like
-specific distributions / variance) can be agreed upon out of band by the
-Aggregators and Collector, there may be benefits to adding explicit protocol
-support by encoding them into task parameters.]
```

Comment:
I deleted this because distribution of task parameters, including DP parameters, is out of scope for DAP.